frontend/src/constants/toasts.ts

```diff
@@ -0,0 +1,16 @@
+import { toast } from 'react-toastify';
+
+/**
+ * The purpose of this file is to centralize the toasts in use in the application in one location. This should allow us to minimize duplication of toast messages using toastIds. https://fkhadra.github.io/react-toastify/prevent-duplicate
+ * In the future we may need to split this file out (at least into /features).
+ */
+
+/** These toasts are used by the update user api */
+export const USER_UPDATING_TOAST_ID = 'UPDATING_USER';
```

Comment:
I like centralization of string constants, but I think you're missing out on an opportunity to namespace this.  Presently this would result in a massive constant list of variables for all different types of toasts.  Find a way to namespace it would result in better maintainability and discoverability with intelisense.

Reply:
The way I was thinking we would namespace this is 
import * as toasts from toasts;
this allows you to treat this entire file as an object, meaning that intellisense works
ie toasts.USER_UPDATING_TOAST_ID

Reply:
Wouldn't it be cleaner to have something like `toasts.user.UPDATING_TOAST_ID`?  Then each file or object would contain their own toasts.

Reply:
sure, I can do that.